tests/add-to-cart.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { test, expect } from '@playwright/test';
+
+
+test('test', async ({ page }) => {
+  await page.goto('https://practicesoftwaretesting.com/');
+  await page.locator('[data-test="product-01JR2SQFT1FHCQQD35GS5JAMHC"]').click();
+  expect(page.url()).toContain('https://practicesoftwaretesting.com/product');
+  await expect(page.locator('[data-test="product-name"]')).toHaveText('Slip Joint Pliers');
```

Comment:
I'd use the getByTestId method to find an element. But you must do this before https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute